frontend/src/pages/TripFormPage.tsx

```diff
@@ -1,15 +1,56 @@
 import "./TripFormPage.css";
+import LocationMap from "../components/LocationMap";
+import AddLocationForm from "../components/AddLocationForm";
+import Trip from "../types/Trip";
+import {useState} from "react";
+import {useNavigate} from "react-router-dom";
+import Api from "../api/Api";
+import Location from "../types/Location";
 
 export default function TripFormPage() {
-    // show input for title
-    // show map with markers for all added locations
-    // show AddLocationForm
-    // save button -> on click -> POST to api
-    // if post successful -> navigation to DashboardPage ?
+
+    const [trips, setTrips] = useState<Trip[]>([]);
+    const [locations, setLocations] = useState<Location[]>([]);
+    const [trip, setTrip] = useState<Trip>({title: "", tripTimestamp: "", locations: []});
```

Comment:
Claudia has taken the parameters out and wrote a separate empty Trip type for this. Yours is not wrote, I would just like to have a discussion to have these things uniform across the project